src/commands/generate/delta.ts

```diff
@@ -84,7 +90,7 @@ export default class GenerateDelta extends BaseCommand<typeof GenerateDelta> {
     }),
     controlsDir: Flags.string({
       char: 'c', required: false, exclusive: ['interactive'],
-      description: 'The InSpec profile directory containing the controls being updated (controls Delta is processing)'}),
+      description: '\x1B[31m(required with -M or -J not provided)\x1B[34m The InSpec profile directory containing the controls being updated (controls Delta is processing)'}),
```

Comment:
```suggestion
      description: '\x1B[31m(required with -M or -J not provided)\x1B[34m The InSpec profile directory containing the controls to update'}),
```

This feels slightly more clear to me, but it is fine either way.